l10n_pt_account/__openerp__.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Contabilidade PT',
+    'version': '1.122',
+    'author': 'ThinkOpen Solutions,Odoo Community Association (OCA)',
+    'category' : 'Accounting & Finance',
+    'sequence': 1,
+    'license' : 'LGPL-3',
+    'description': '''Customization of Accounting for Portugal.
+This module implements several Portuguese specific accounting documents:
+    Documents:
+        * Debit Notes; 
+        * Credit Notes;
+        * Supplier Debit Notes;
+        * All Waybill Types.
+        
+    Reports:
+        * Voucher Print
+        
+    * Allows you to import products, clients and invoices to saft
+NOTE: This documents to be valid, need a certification from Portuguese Treasury, for that you must install and configure the tko_ics2_pt module, provided by Thinkopen Solutions.''',
+    'website': 'http://www.thinkopen.solutions/',
+    'init_xml': [],
+    'depends': ['base',
+                'account',
+                'account_cancel',
+                'account_voucher',
+                'purchase',
+                'sale',
+                'stock',
+                'stock_account',
```

Comment:
Can this module be split in smaller ones? Purchase related features depend on purchase, sale features depend on sale, etc...